www/templates/default/html/EventInstance/OriginCalendar.tpl.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use UNL\UCBCN\Frontend\Controller;
+
+$originCalendar = $context->event->getOriginCalendar();
+if (!empty($context->calendar->id) && !empty($originCalendar->id) && $context->calendar->id != $originCalendar->id): ?>
+    <?php
+        $name = $savvy->dbStringtoHtml($originCalendar->name);
+        $name = $savvy->linkify($name);
+    ?>
+    <div class="dcf-mt-6 dcf-txt-2xs">
```

Comment:
I suggest changing this to:
`<div><small class="dcf-txt-2xs">This event originated in <a href="<?php echo Controller::$url . urlencode($originCalendar->shortname); ?>"><?php echo $name; ?></a>.</small></div>`